### ISSUE TITLE
Make it clear where extensions come from

### ISSUE THREAD

**Issue by jerabaul29** [onivim/oni2#2434]
Would it be possible to make it clear where extensions come from, for example by putting the link to the github repo at the start of the extension page? For example, I want to have to python tool. From this:

![Screenshot from 2020-09-11 16-58-31](https://user-images.githubusercontent.com/8382834/92940943-08c65c80-f450-11ea-83e9-ebd13e98ac54.png)

I suppose this is the "official" VSC extension, right? Any way I can be sure of it? Could you add some way to easily understand who the author is?

**Comment by bryphe** [comment]
Good point - we should put more context here to help validate the source.

Also related to #2345
